### DISCUSSION TITLE
Hidden Commits

### DISCUSSION THREAD

**Discussion by rcalixte** [linuxmint/discussions#38] [Development]
**Proposal**
Following the GitHub documentation [here](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view), adding `.git-blame-ignore-revs` to repos with commits that should be hidden by default in the blame view. This leaves room for future commits for code cleanup that will also not detract from the utility of the blame view.

Example pull request:
* linuxmint/cinnamon#11700

**Comment by clefebvre** [comment]
That's a pretty interesting feature.
